tests/test_dataframes.py

```diff
@@ -223,6 +223,238 @@ def test_on_dataframes_with_object_columns_with_nans(self):
         )[0]
 
```

Comment:
It would be good to apply the old `TestAreDataFramesEqual` on the new `are_equal` method.

Reply:
Great idea - did that!